conbench/templates/index.html

```diff
@@ -119,8 +119,8 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
           "searchPlaceholder": "commit hash works too",
       }
 });
-
 enable_search_query_string('#runs');
+specific_column_search('#runs', [3,6]);
```

Comment:
Unfortunately the Arrow conbench instance has many more hardware names than our repo's dummy data -- I just counted 68 in the last 1000 runs:
```
>>> from benchclients import ConbenchClient
>>> c = ConbenchClient()
>>> runs = c.get('/runs')
>>> len(set(run['hardware']['name'] for run in runs))
68
```
I think this will be too many for a dropdown. Instead of hardcoding which columns to apply the dropdown to, I wonder if it would be better to automatically apply it to any column that has less than, for example, 10 unique values? That might be tricky to code but could be worth it for the UX.

Alternatively, you could just get rid of the dropdown for the Hardware column. It will probably still be useful for Reason and benchmark Language.

Reply:
Or maybe in circumstances when there are many unique values, we switch over to a free-text search. We don't have to do that in this PR, but it would be nice to add that for columns that have many unique values IMHO

Reply:
> Unfortunately the [Arrow conbench instance](https://conbench.ursa.dev/) has many more hardware names than our repo's dummy data -- I just counted 68 in the last 1000 runs

This is my fault and needs cleaning up; I fixed the C++ microbenchmark hardware name issue, but I think the Go ones are still problematic, so I haven't cleaned up the db yet. The real number should be [7-8](https://github.com/voltrondata-labs/arrow-benchmarks-ci/blob/main/config.py#L65) (Go is still separate for now).